CMakeLists.txt

```diff
@@ -1485,6 +1494,7 @@ if (NOT EVENT__DISABLE_TESTS)
             "
             set EVENT_NOWIN32=
             set EVENT_NOWEPOLL=
+            set EVENT_NOPSN=
```

Comment:
It is supported only under win32, right? (i.e. not in mingw build, so we don't need to update https://github.com/libevent/libevent/blob/66861f8891ef8dee453fb5500376d2e0b6ff8092/test/test.sh#L3)